TwilioVerifyDemo/TwilioVerifyDemo/Settings/SettingsViewController.swift

```diff
@@ -0,0 +1,135 @@
+//
+//  SettingsViewController.swift
+//  TwilioVerifyDemo
+//
+//  Created by Alejandro Orozco Builes on 16/12/21.
+//  Copyright © 2021 Twilio. All rights reserved.
+//
+
+import UIKit
+import TwilioVerifySDK
+
+class SettingsViewController: UIViewController, UITableViewDataSource, UITableViewDelegate {
+  
+  let storage = SecureStorage(accessGroup: nil)
+
+  enum AppSetting: String, CaseIterable {
+    case clearStorage
+    case currentVersion
+    case useAccessGroup
+
+    var description: String {
+      switch self {
+        case .clearStorage:
+          return "Clear Storage"
+        case .currentVersion:
+          return "Storage Version"
+      case .useAccessGroup:
```

Comment:
<table data-meta="generated_by_UnitTests">
  <tbody>
    <tr>
      <td>:warning:</td>
      <td width="100%" data-sticky="false"><span data-href="https://github.com/twilio/twilio-verify-ios/blob/b791acb3cd9efbf75408116c9fe627d5a3319f09/TwilioVerifyDemo/TwilioVerifyDemo/Settings/SettingsViewController.swift#L27"></span>Case statements should be indented within their enclosing switch statement.<br />
<code>switch_case_alignment</code> <code>SettingsViewController.swift:27</code></td>
    </tr>
  </tbody>
</table>